src/compiler/diagnosticMessages.json

```diff
@@ -3310,6 +3310,10 @@
         "category": "Error",
         "code": 7034
     },
+    "Try `npm install @types/{0}` if it exists or adding `declare module '{0}'` in a separate file.": {

```

Comment:
I agree with @bterlson, technically any non-module file would work, but it is clearer to say `in a separate .d.ts file.`